MAINTAINERS.md

```diff
@@ -0,0 +1,144 @@
+  * The `proto/spire/vX.X.X` tag is needed for proper versioning of the github.com/spiffe/spire/proto/spire go module and can be omitted if/when that go module is no longer in the SPIRE repository.
+* Push the tags to SPIRE, and watch the build to completion
+  * If the build fails, or anything unusual is encountered, abort the release
+    * Ensure that the GitHub release, container images, and release artifacts are deleted/rolled back if necessary
+* Visit the releases page on GitHub, copy the release notes, click edit and paste them back in. This works around a GitHub rendering bug that you will notice before completing this task
+* Open and merge a PR to bump the SPIRE version to the next projected version and apply the CHANGELOG updates
+  * For example, after releasing 0.10.0, update the version to 0.10.1, since it is more likely to be released before 0.11.0.
+  * Ideally, this is the first commit merged following the release
+
+**IF this is a major release**, the following steps must be completed no later than one week after the release:
+* PRs to update spiffe.io and spire-examples repo to the latest major version must be merged
+  * Ensure that the PRs have been updated to use the version tag instead of the commit sha
+* Broadcast news of release to the community via available means: SPIFFE Slack, Twitter, etc.
+
+## Community Interaction and Presence
+Maintainers represent the front line of SPIFFE and SPIRE community engagement. They are the ones interacting with end users in issues, and with contributors on their PRs. The buck stops with a maintainer.
+
+SPIRE maintainers must make themselves available to the community. It is critical that maintainers engage in this capacity - for understanding user needs and pains, for ensuring success in project adoption and deployment, and to close feedback loops on recently-introduced changes or features... to name a few.
+
+PR and Issue management/response is table stakes for all SPIRE maintainers. In addition, maintainers should, whenever possible:
+* Be generally available on the SPIFFE Slack, and engage in questions/conversations raised in the #help and #spire channels
+* Attend SPIFFE/SPIRE community events (physically or virtually)
+* Present SPIFFE/SPIRE at meetups and industry conferences
+
+### Communication Values
+SPIRE maintainers always engage in a respectful and constructive manner, and always follow the [SPIFFE Code of Conduct][6].
+
+It is very important for maintainers to understand that contributions are generally acts of kindness, whether it be creating an issue or sending a pull request. It takes time to do these things. In the vast majority of cases, the motivating factor for taking the time to do this is either to improve the quality of the project for others, or to enable the project to (more easily?) solve a problem that it could not previously. Both of these factors are positive.
```

Comment:
```suggestion
It is very important for maintainers to understand that contributions are generally acts of generosity, whether it be creating an issue or sending a pull request. It takes time to do these things. In the vast majority of cases, the motivating factor for taking the time to do this is either to improve the quality of the project for others, or to enable the project to (more easily?) solve a problem that it could not previously. Both of these factors are positive.
```